docs/specifications/tests/Consistency-TP/consistency01.md

```diff
@@ -33,27 +33,37 @@ serial number consistency.
 
  1. Obtain the list of name server IPs for the *child zone* from [Method4] 
     and [Method5].
+
  2. Create an SOA query for *child zone* apex and send it to all name 
     server IPs.
+
  3. Retrieve the SOA RR from the responses from all name server IPs.
```

Comment:
Wouid it be possible to more this step down until after we've checked which responses have SOA RR sets? I think the flow of the TC would be improved that way.
Also, why do we extract the entire SOA RR and not just the SOA/SERIAL?